src/layers.cpp

```diff
@@ -105,15 +111,15 @@ Layer_::updateActiveLayers(void) {
       int8_t layer = highestLayer;
 
       while (layer > DefaultLayer) {
-        if (Layer.isOn(layer)) {
+        if (isOn(layer) || layer == highestLayer) {
           Key mappedKey = (*getKey)(layer, row, col);
 
           if (mappedKey != Key_Transparent) {
             activeLayers[row][col] = layer;
             break;
           }
         }
-        layer--;
+        --layer;
```

Comment:
Oops. I changed this decrement to a prefix operator with the thought that it would avoid an unnecessary copy, but of course, the compiler already optimized that out. It shouldn't really be part of this change, but it's also harmless.